spec/registry.md

```diff
@@ -833,3 +818,25 @@ For more information, see [Working with Timezones](https://w3c.github.io/timezon
 > The form of these serializations is known and is a de facto standard.
 > Support for these extensions is expected to be required in the post-tech preview.
 > See: https://datatracker.ietf.org/doc/draft-ietf-sedate-datetime-extended/
+
+### Date and Time Locale Options
+
+**_<dfn>Date/time locale options</dfn>_** override options set by the current locale
+or an implementation-defined date/time _operand_ value.
+
+The following options and their values are required to be available on
+the functions `:datetime`, `:date`, and `:time`:
+
+- `hour12`
+  - `true`
+  - `false`
+- `calendar` \[OPTIONAL\]
+  - valid [Unicode Calendar Identifier](https://cldr-smoke.unicode.org/spec/main/ldml/tr35.html#UnicodeCalendarIdentifier)
+- `numberingSystem` \[OPTIONAL\]
+  - valid [Unicode Number System Identifier](https://cldr-smoke.unicode.org/spec/main/ldml/tr35.html#UnicodeNumberSystemIdentifier)
+- `timeZone` \[OPTIONAL\]
+  - valid identifier per [BCP175](https://www.rfc-editor.org/rfc/rfc6557)
```

Comment:
Each of the optional options I think should have a value space of "valid identifier per [something] or an implementation-defined value"

---

It is important to be able to set `none` or blank as an option, as a way to allow developers to "float" a date/time value. See [floating-time](https://w3c.github.io/timezone/#dfn-float)